tests/fix_pq.py

```diff
@@ -38,7 +40,10 @@ def libpq():
 
     try:
         # Not available when testing the binary package
-        libname = ctypes.util.find_library("pq")
+        if sys.platform == "win32":
+            libname = ctypes.util.find_library("libpq.dll")
+        else:
+            libname = ctypes.util.find_library("libpq")
```

Comment:
I guess this has to be `"pq"`

Reply:
I've force-pushed a fix for this typo (in both places).